Composer/packages/client/src/components/AppComponents/RightPanel.tsx

```diff
@@ -5,12 +5,21 @@
 import { jsx, css } from '@emotion/core';
 import { useRecoilValue } from 'recoil';
 import { forwardRef } from 'react';
+// import formatMessage from 'format-message';
 
 import { RequireAuth } from '../RequireAuth';
 import { ErrorBoundary } from '../ErrorBoundary';
+import { Conversation } from '../Conversation';
+//import { ProjectTree } from '../ProjectTree/ProjectTree';
+//import { LeftRightSplit } from '../Split/LeftRightSplit';
```

Comment:
Can remove comments

Reply:
In this case, I don't want to. These are there to remind me where the stubs are for future work.